### ISSUE TITLE
Player patch version not matching 

### ISSUE THREAD

**Issue by Abbyh800** [an-anime-team/the-honkers-railway-launcher#150]
Player patch version is 3.1.3, game is now 2.3 
How do I get the launcher to match and play the game? Was super excited for the new update but now I can’t access the game. I updated the launcher which prompted the update of the game itself. Now I can’t get it to launch.

**Comment by Chakrabortysoura** [comment]
yeah facing the same issue. warning about unitylauncher.dll checksum is the issue it seems.
![Screenshot from 2024-06-19 08-04-25](https://github.com/an-anime-team/the-honkers-railway-launcher/assets/95709386/15a62e97-c423-41fd-b294-7e9a9af4df50)

**Comment by swwind** [comment]
https://codeberg.org/mkrsym1/jadeite/issues/53

**Comment by Chakrabortysoura** [comment]
@swwind yeah got it but in comment someone mentioned using something to launch the game but how to do that?

**Comment by swwind** [comment]
> @swwind yeah got it but in comment someone mentioned using something to launch the game but how to do that? I referring to SCGM's comment about using BREAK_CRYPTCAT=1

Just add this when starting `the-honkers-railway-launcher`. like

`BREAK_CRYPTCAT=1 the-honkers-railway-launcher`

**Comment by RedPenguin88** [comment]
Note that according to [this](https://codeberg.org/mkrsym1/jadeite/issues/37), using that variable may result in a ban, though that issue was made 5 months ago, so not sure if still true.

**Comment by Zylade** [comment]
> > @swwind yeah got it but in comment someone mentioned using something to launch the game but how to do that? I referring to SCGM's comment about using BREAK_CRYPTCAT=1
> 
> Just add this when starting `the-honkers-railway-launcher`. like
> 
> `BREAK_CRYPTCAT=1 the-honkers-railway-launcher`

How do I use this?

**Comment by ghostlyentity** [comment]
> > @swwind yeah got it but in comment someone mentioned using something to launch the game but how to do that? I referring to SCGM's comment about using BREAK_CRYPTCAT=1
> 
> Just add this when starting `the-honkers-railway-launcher`. like
> 
> `BREAK_CRYPTCAT=1 the-honkers-railway-launcher`
![Screenshot from 2024-06-19 06-12-44](https://github.com/an-anime-team/the-honkers-railway-launcher/assets/86599520/18866d42-4e4e-44c4-8e13-e5976fe816e6)


doesnt work for me, it loops back to the launcher

**Comment by athena11778** [comment]
> > > @swwind yeah got it but in comment someone mentioned using something to launch the game but how to do that? I referring to SCGM's comment about using BREAK_CRYPTCAT=1
> > 
> > 
> > Just add this when starting `the-honkers-railway-launcher`. like
> > `BREAK_CRYPTCAT=1 the-honkers-railway-launcher`
> > ![Screenshot from 2024-06-19 06-12-44](https://private-user-images.githubusercontent.com/86599520/340909776-18866d42-4e4e-44c4-8e13-e5976fe816e6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3NzA4NTUsIm5iZiI6MTcxODc3MDU1NSwicGF0aCI6Ii84NjU5OTUyMC8zNDA5MDk3NzYtMTg4NjZkNDItNGU0ZS00NGM0LThlMTMtZTU5NzZmZTgxNmU2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE5VDA0MTU1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRjNjQ2YzBmYzM2YmI1ODZlMWFjNDgyMTIzOGFlNzlmMjdmYTY3Mzc0ZDE0OTY4MWY1NTRjNzZiZGE5MTQ3ZDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.O3Ds46C97KVh3W7EWAkY3iBUDKIRqMToT2Vqg_FOLjI)
> 
> doesnt work for me, it loops back to the launcher

same :(

**Comment by Chrisss88** [comment]
Is there any other way run this game?
I wanna play this so badly on my steamdeck

**Comment by pd90506** [comment]
In the launcher -> settings icon -> enhancements -> Environment -> In the New variable section, set Name: BREAK_CRYPTCAT and Value: 1. This will work.

**Comment by Chrisss88** [comment]
> In the launcher -> settings icon -> enhancements -> Environment -> In the New variable section, set Name: BREAK_CRYPTCAT and Value: 1. This will work.

![Screenshot_20240619_125707](https://github.com/an-anime-team/the-honkers-railway-launcher/assets/173225479/ef2bff1d-5e94-4891-b4f8-2a9a35691d18)

still cant get in

**Comment by ghostlyentity** [comment]
> > In the launcher -> settings icon -> enhancements -> Environment -> In the New variable section, set Name: BREAK_CRYPTCAT and Value: 1. This will work.
> 
> ![Screenshot_20240619_125707](https://private-user-images.githubusercontent.com/173225479/340941775-ef2bff1d-5e94-4891-b4f8-2a9a35691d18.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3NzcxMTQsIm5iZiI6MTcxODc3NjgxNCwicGF0aCI6Ii8xNzMyMjU0NzkvMzQwOTQxNzc1LWVmMmJmZjFkLTVlOTQtNDg5MS1iNGY4LTJhOWEzNTY5MWQxOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxOVQwNjAwMTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xYmI5Zjg5YWJiNDdhYWNhYmUyNjI1NWIwNjQzYzg2NzE4MmEyMWIyNDQ5OWNmNjQ5MmJjMjlhYTRmZDJkNDhiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.iGO0x0DT-3-i1WMZk40knJWc_WOrFhWFu18e8-YKiLc)
> 
> still cant get in

i concur

**Comment by EvilBoi123** [comment]
oh well, i suppose for now, i have to copy the game folder into windows machine temporary until this issue resolve then.

**Comment by scgm0** [comment]
> > In the launcher -> settings icon -> enhancements -> Environment -> In the New variable section, set Name: BREAK_CRYPTCAT and Value: 1. This will work.
> 
> ![Screenshot_20240619_125707](https://private-user-images.githubusercontent.com/173225479/340941775-ef2bff1d-5e94-4891-b4f8-2a9a35691d18.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3OTk4NDEsIm5iZiI6MTcxODc5OTU0MSwicGF0aCI6Ii8xNzMyMjU0NzkvMzQwOTQxNzc1LWVmMmJmZjFkLTVlOTQtNDg5MS1iNGY4LTJhOWEzNTY5MWQxOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxOVQxMjE5MDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lZmIyNzIyZWU4Y2YwNTVjYTc4NTI3Yzk4ZDEzYmMzOWIxMmQ0YWM3MWQxOGFmODM5ZTU4NjE4MGNmNThiMDhiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.92cSKLvo9FsndqMFFNkauuCXGMWQNsUWjMUEKSaBSfw)
> 
> still cant get in

For me, the game starts up normally after clicking the `OK` button.
![图片](https://github.com/an-anime-team/the-honkers-railway-launcher/assets/52148221/68a6d494-8580-4cb9-b01c-3261949412a9)

**Comment by Dvd-Znf** [comment]
@scgm0 it only works like that for CN version, check patch repo or discord

**Comment by Chakrabortysoura** [comment]
it seems they patched it when i opened the launcher now it told me to download a patch it is working fine now

**Comment by Chrisss88** [comment]
> it seems they patched it when i opened the launcher now it told me to download a patch it is working fine now

Thanks for your information bro
Mine still downloading 6gb patch in game now

**Comment by athena11778** [comment]
yup, after the patch the game is working for me with no adjustments !

**Comment by lovelylucina** [comment]
Did the patch auto-prompt for you all? mine is noting test patch available but not actually downloading

**Comment by ghostlyentity** [comment]
can confirm, the game window launches now

**Comment by Serpentian** [comment]
It launches, but version mismatch is still there:

![image](https://github.com/an-anime-team/the-honkers-railway-launcher/assets/106916212/fa683a1a-16c4-46c9-aae1-a14724885e34)
